### PR TITLE
Feat/Added Blog Page Layout

### PR DESCRIPTION
**Pull request by zClerigo**
Screenshot 2024-10-14 at 21-58-11 Stevens Blueprint Blog

- This layout does not interact with the files in the blog folder
- Screenshot displays the page when the code in blog.md is uncommented.
- Blog page currently displays placeholder layout

### CODE REVIEW

**Comment by zClerigo** [comment]
I am not sure why blog.md is failing prettier check.

**Comment by Caposto** [comment]
@zClerigo Looking good so far! In regards to the blog failing prettier, what happens when you run `npm run prettier:write`?

File: src/assets/vector/clock.png
Comment (Caposto) [inline src/assets/vector/clock.png:1]: Is there a .svg version of the clock? Not a huge deal, but svg files are typically smaller and can scale well without blurring

File: src/blog.md
Comment (Caposto) [inline src/blog.md:1]: Prettier might be failing due to the extra space at the bottom, but lmk what happens when you run `npm run format:write`

File: src/_includes/_layouts/Blog.jsx
Comment (Caposto) [inline src/_includes/_layouts/Blog.jsx:1]: Great work so far! I know you're still working, but noticed there's a bit of padding on the left and right sizes. I think in the designs we might want to be a little closer to the edges:

image

I think running throgh and adjusting some of the mx-*, mb-*, p-* could do the trick cc @ShawnAviles

File: src/assets/vector/clock.png
Comment (zClerigo) [reply src/assets/vector/clock.png:1]: Hi, Caposto. Running `npm run prettier:write` results in the blog.md file being formatted smoothly.

I also replaced the .png with a .svg clock. I will edit this pull request promptly.

**Review by Caposto — CHANGES_REQUESTED** [review]
- For the svgs for filter topics add them to the assets folder as well
- Adjust padding to fill more of the page
- Add 3 more example blogs

File: src/_includes/_layouts/Blog.jsx [hunk 1]
```diff
@@ -10,23 +10,141 @@ export default ({ comp, description, posts, children }) => (
     <body>
       <comp.Navbar />
       <main className="max-w-4xl mx-auto p-4">
```
Comment (ShawnAviles) [inline src/_includes/_layouts/Blog.jsx]: Adding onto Christian's comment about the padding on the left and right, here the width is being maxxed out at `max-w-4xl` but that is what's restricting the layout to have such large left and right padding. 

You can change it be something like `max-w-6xl` to reduce the large horizontal space but other things would have to increase in size too to accommodate the white space

**Review by ShawnAviles — CHANGES_REQUESTED** [review]
Left comment about the large white space on the left and right but overall nice job! Love how similar you got it to the designs 🫡

**Comment by zClerigo** [comment]
Screenshot 2024-10-17 at 19-14-13 Stevens Blueprint Blog
New changes pushed. I did not need to add an svg file in the assets folder because the svg and path elements draw out the dropdown arrow shape (taken from tailwind documentation).

**Review by ShawnAviles — DISMISSED** [review]
Nice job, LGTM 👍 

NIT: Just so we know, it would be helpful to have a comment there stating that "Uncomment this to show mock blog posts" so that we can remember incase this PR gets lost or someone without context were to try to view it.

**Review by Caposto — APPROVED** [review]

**Pull request merged**